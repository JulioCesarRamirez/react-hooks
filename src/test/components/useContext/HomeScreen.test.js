import React from 'react';
import '@testing-library/jest-dom';
import { mount } from 'enzyme';
import { HomeScreen } from '../../../components/useContext/HomeScreen';
import { UserContext } from '../../../components/useContext/UserContext';

describe('Testing <HomeScreen />', () => {
  const user = {
    name: 'Fernando',
    email: 'fernando@gmail.com',
  };
  const wrapper = mount(
    <UserContext.Provider value={{user}}>
      <HomeScreen />
    </UserContext.Provider>
  );

  it('should render correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
