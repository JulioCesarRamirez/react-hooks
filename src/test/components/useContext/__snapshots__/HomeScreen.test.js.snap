// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing <HomeScreen /> should render correctly 1`] = `
<div>
  <h1>
    Home screen
  </h1>
  <hr />
  <pre>
    {
   "name": "Fernando",
   "email": "fernando@gmail.com"
}
  </pre>
</div>
`;
